packages/transform/src/useTransformHandler.ts

```diff
@@ -23,25 +23,25 @@ import { useEffect } from '@googleforcreators/react';
  * Internal dependencies
  */
 import useTransform from './useTransform';
+import type { TransformHandler } from './types';
 
 /**
- * @callback TransformHandler
- * @param {?Object} frameObject
- */
-
-/**
- * @param {string} id Target element's id.
- * @param {TransformHandler} handler The transform handler. The argument is
+ * @param id Target element's id.
+ * @param handler The transform handler. The argument is
  * the frame object. The `null` value resets the transform.
- * @param {Array} [deps] The effect's dependencies.
+ * @param [deps] The effect's dependencies.
  */
-function useTransformHandler(id, handler, deps = undefined) {
+function useTransformHandler(
+  id: string,
+  handler: TransformHandler,
+  deps: unknown[] | undefined = undefined
```

Comment:
In other places I've used the `DependencyList` type from the React package for this, e.g.

```suggestion
  deps: DependencyList | undefined = undefined
```